### PR TITLE
[DCOS-54752] Add DSS volume profile support

### PR DESCRIPTION
**Pull request by ishanchandrajoshi**
This branch contains the code which supports adding DSS volume profiles to Hdfs framework.

### CODE REVIEW

File: frameworks/hdfs/universe/config.json [hunk 1]
```diff
@@ -192,6 +192,10 @@
           ],
           "default": "ROOT"
         },
+        "volume_profile": {
+          "type": "string",
+          "description": "Volume profile to be used for storing Journal data."
```
Comment (mpereira) [inline frameworks/hdfs/universe/config.json]:
```suggestion
          "description": "Volume profile to be used for storing JournalNode data."
```

File: frameworks/hdfs/universe/config.json [hunk 2]
```diff
@@ -295,6 +299,10 @@
           "type": "string",
           "default": "ROOT"
         },
+        "volume_profile": {
+          "type": "string",
+          "description": "Volume profile to be used for storing Name node meta-data."
```
Comment (mpereira) [inline frameworks/hdfs/universe/config.json]:
```suggestion
          "description": "Volume profile to be used for storing NameNode data."
```

File: frameworks/hdfs/universe/config.json [hunk 3]
```diff
@@ -787,6 +795,10 @@
           "type": "string",
           "default": "ROOT"
         },
+        "volume_profile": {
+          "type": "string",
+          "description": "Volume profile to be used for storing Cassandra data."
```
Comment (mpereira) [inline frameworks/hdfs/universe/config.json]:
```suggestion
          "description": "Volume profile to be used for storing DataNode data."
```

**Comment by ishanchandrajoshi** [comment]
> Looks good now, thanks @ishanchandrajoshi.

Thank you as well @mpereira

**Comment by samvantran** [comment]
thanks @ishanchandrajoshi changes look good. I also edited the PR title. In the future can you follow a similar format? e.g.
changed from
```
Dcos 54752 dss volume profile issue
``` 
to
```
[DCOS-54752] Add DSS volume profile support
```

**Comment by ishanchandrajoshi** [comment]
> thanks @ishanchandrajoshi changes look good. I also edited the PR title. In the future can you follow a similar format? e.g.
> changed from
> 
> ```
> Dcos 54752 dss volume profile issue
> ```
> 
> to
> 
> ```
> [DCOS-54752] Add DSS volume profile support
> ```

Alright, I am merging this PR.

**Pull request merged**